src/electron/platform/android/setup/steps/configuring-port-forwarding.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { AndroidSetupStepConfig } from 'electron/platform/android/setup/android-setup-steps-configs';
+
+export const configuringPortForwarding: AndroidSetupStepConfig = deps => ({
+    actions: {},
```

Comment:
Should have a cancel handler to go back to `detect-devices`, probably

Reply:
Per offline discussion, leaving for a separate PR (other steps also missing them)